doc/contributing/coding-conventions.chapter.md

```diff
@@ -224,6 +224,8 @@ There are a few naming guidelines:
 
 Example: Given a project had its latest releases `2.2` in November 2021, and `3.0` in January 2022, a commit authored on March 15, 2022 for an upcoming bugfix release `2.2.1` would have `version = "2.2-unstable-2022-03-15"`.
 
+(Reviewers: you can often check that a package's unstable version is correct, or get a suggestion for what to use instead, by running `maintainers/scripts/check-unstable-version.sh $PR_NUMBER`. This script isn't foolproof; common sense and this manual take precedence.)
```

Comment:
Please link to that file on GitHub, otherwise one has to do manual steps to find out details.